sdk/iot/hub/tests/cmocka/test_az_iot_hub_client_sas.c

```diff
@@ -184,15 +185,12 @@ static void az_iot_sas_token_get_document_succeeds(void** state)
   int32_t expiry_time_secs = TEST_EXPIRATION;
 
   uint8_t raw_document[TEST_SPAN_BUFFER_SIZE];
-  memset(raw_document, 0xFF, _az_COUNTOF(raw_document));
-  az_span document = az_span_init(raw_document, 0, _az_COUNTOF(raw_document));
+  az_span document = az_span_for_test_init(raw_document, 0, _az_COUNTOF(raw_document));
 
   assert_true(az_succeeded(az_iot_sas_token_get_document(
       iothub_fqdn, device_id, expiry_time_secs, document, &document)));
-  assert_memory_equal(az_span_ptr(document), expected_document, strlen(expected_document));
-  assert_int_equal(az_span_length(document), strlen(expected_document));
-  assert_int_equal(az_span_capacity(document), TEST_SPAN_BUFFER_SIZE);
-  assert_int_equal(raw_document[az_span_length(document)], 0xFF);
+  az_span_for_test_verify(
```

Comment:
Nit: (Pre-existing): the expected_document here and other functions in this sas test, to be consistent with other tests, should be globals defined at top of file?

Reply:
Nit2: This would also suggest a sizeof(…) instead of strlen's.

Reply:
agreed I will look to update this in another PR